GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -312,7 +311,18 @@ public void beforeDrawEyes() {
                 getMainScene().addSceneObject(mSplashScreen);
             }
 
-            mScript.onInit(GVRViewManager.this);
+            try {
+                mScript.onInit(GVRViewManager.this);
+            } catch (Throwable t) {
+                t.printStackTrace();
```

Comment:
I am thinking if you don't want to crash the app at least we need to print the error message to the console with any reasonable tag.  I was thinking printStackTrace doesn't show anything in logcat and I just tried and that it is. So if there is any kind of exception occurs in onInit, there is just quick quit of the app but no error message shown to developers. I think it may make developers a little bit confused and hard to find any clue to debug.

I am considering it is better to the getStackTrace() to get stack trace info and use Log.e to print such information.